app/src/main/java/com/example/hungrywolfs/model/DetailsViewModel.kt

```diff
@@ -0,0 +1,57 @@
+package com.example.hungrywolfs.model
+
+import android.util.Log
+import android.view.View
+import android.widget.ImageButton
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.example.hungrywolfs.R
+import com.example.hungrywolfs.SingleLiveEvent
+import com.example.hungrywolfs.network.MealApi
+import com.example.hungrywolfs.network.MealDetail
+import kotlinx.coroutines.launch
+
+class DetailsViewModel : ViewModel() {
+
+    private val _navigationToHome = SingleLiveEvent<Any>()
+    val navigationToHome = _navigationToHome
+
+    private val _mealDetails = MutableLiveData<MealDetail>()
```

Comment:
Please update this LiveData to be of type MealDetailInfo since there's only one item and the only one we should care about here.